components/home/IntegrationInstallation.tsx

```diff
@@ -1,6 +1,6 @@
 import { Container, Row, Col } from "react-bootstrap";
 import Link from "next/link";
-import HomepageCodeBox from "../../components/home/homepageCodeBox";
+import HomepageCodeBox from "./HomepageCodeBox";
```

Comment:
nice - some simpler imports

Reply:
That fixed itself when I let VSCode auto-correct the import.  Thanks VSCode!